packages/server/src/login/login.controller.ts

```diff
@@ -169,7 +103,11 @@ export class LoginController {
 
   // Set cookie and redirect to proper page
   private async enter(res: Response, userId: number) {
-    const authToken = await this.jwtService.signAsync({ userId });
+    // Expires in a day
+    const authToken = await this.jwtService.signAsync({
+      userId,
+      expiresIn: 60 * 60 * 24 * 30,
+    });
```

Comment:
Does this expire in a day or a month? A day seems pretty short

Reply:
good catch. 30 days